internal/tools/tools.go

```diff
@@ -23,7 +23,6 @@ import (
 	_ "github.com/golangci/golangci-lint/cmd/golangci-lint"
 	_ "github.com/google/addlicense"
 	_ "github.com/jstemmer/go-junit-report"
-	_ "github.com/ory/go-acc"
```

Comment:
This was the tool we were previously using for code coverage. The [project](https://github.com/ory/go-acc) has been archived.

Reply:
Adding this package back in to reduce PR's scope